org-cyf-itd/content/steps/zero/index.md

```diff
@@ -5,7 +5,7 @@ layout = 'prep'
 emoji= '🗨'
 menu_level = ['module']
 menu=[ 'steps']
-weight = 1
+weight = 0
```

Comment:
You will have to run through all the steps and update the weight so this should be 1, 1 is 2, 3 is 4... etc.

The step 0 addition was a curveball!